### PR TITLE
Update OCMock version to 3.5 in RC podspec.

### LINKED ISSUE #4348: Fix RemoteConfig tests for OCMock 3.5.0
https://travis-ci.org/firebase/firebase-ios-sdk/jobs/613701585:

    - NOTE  | [iOS] xcodebuild:  /Users/travis/build/firebase/firebase-ios-sdk/FirebaseRemoteConfig/Tests/Unit/RCNConfigContentTest.m:58: error: -[RCNConfigContentTest testCrashShouldNotHappenWithoutMainBundleID] : failed: caught "NSInternalInconsistencyException", "Mock object was not used in OCMStub/OCMExpect/OCMReject. Did you accidentally use a real object?"

### PR DESCRIPTION
**Pull request by dmandar**
Fix #4348

### CODE REVIEW

File: FirebaseRemoteConfig.podspec [hunk 1]
```diff
@@ -68,6 +68,6 @@ app update.
         'FirebaseRemoteConfig/Tests/Unit/Defaults-testInfo.plist',
         'FirebaseRemoteConfig/Tests/Unit/SecondApp-GoogleService-Info.plist'
     unit_tests.requires_app_host = true
-    unit_tests.dependency 'OCMock', '~> 3.4.0'
+    unit_tests.dependency 'OCMock', '~> 3.5.0'
```
Comment (paulb777) [inline FirebaseRemoteConfig.podspec]: Consider no version here and letting the dependency float like the podspecs.

If something breaks when OCMock updates, it doesn't impact the product and we probably want to know about it.

File: FirebaseRemoteConfig/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+$ v4.4.8
+- [changed] Update OCMock version dependency to v3.5 in Remote Config podspec.
```
Comment (paulb777) [inline FirebaseRemoteConfig/CHANGELOG.md]: CHANGELOG not necessary since the change only impacts unit tests and not product.

Comment (dmandar) [reply FirebaseRemoteConfig/CHANGELOG.md]: sg.

File: FirebaseRemoteConfig.podspec (hunk 1 above)
Comment (dmandar) [reply FirebaseRemoteConfig.podspec]: sg. Done.

**Pull request merged**